src/native/libs/CMakeLists.txt

```diff
@@ -58,6 +58,12 @@ if (CLR_CMAKE_TARGET_UNIX OR CLR_CMAKE_TARGET_BROWSER OR CLR_CMAKE_TARGET_WASI)
         if (CROSS_ROOTFS)
             include_directories(SYSTEM "${CROSS_ROOTFS}/usr/include")
         endif ()
+
+        # Google requires all the native libraries to be aligned to 16 bytes (for 16k memory page size)
```

Comment:
This should be set under https://github.com/dotnet/runtime/tree/main/eng so that it applies to all binaries that target Android now and in future.